test/empty.fromasm.clamp.no-opts

```diff
@@ -1,6 +1,6 @@
 (module
  (import "env" "memory" (memory $0 256 256))
  (import "env" "table" (table 0 0 anyfunc))
- (import "env" "memoryBase" (global $memoryBase i32))
- (import "env" "tableBase" (global $tableBase i32))
+ (import "env" "memoryBase" (global $memoryBase (mut i32)))
```

Comment:
@kripken It's not clear why all these changed to mutable. Perhaps this is revealing a bug I introduced?

Reply:
Oh! Is it because those asmjs globals were actually mutable but previously we couldn’t represent that?

Reply:
Interesting - I think that's a coincidence actually, but a good one here as it pointed out something useful :) What I think is going on is that we forgot an initial value for `bool mutable_;` (unlike the other fields near it, it doesn't have a default value) - we should set that to `bool mutable_ = false;`.

It's also a good question what we should do for asm.js imports. Yeah, in asm.js they are mutable, and to work around that in wasm we created a mutable global with the initial value of the import, then we just use that global from then on. In theory we could simplify that to use a mutable import, however, I think it would be more complicated than it would be worth since we do want the old form supported too (and also a mutable import may have perf overhead).